app/controllers/providers/proceedings_types_controller.rb

```diff
@@ -25,7 +25,10 @@ def create
     private
 
     def run_transaction
-      # TODO: Should be fixed by future LFA ticket to change to Proceeding record
+      proceeding_to_add = proceeding_types.find { |proceeding| proceeding.ccms_code == form_params }
+      add_proceeding_service.call(ccms_code: proceeding_to_add.ccms_code)
```

Comment:
I thought the idea of this ticket was to create the Proceeding record directlly, and not rely on the ProceedingTypes